ignition-rndf.rb

```diff
@@ -0,0 +1,30 @@
+class IgnitionRndf < Formula
+  desc "Ignition RNDF is a portable C++ library for parsing RNDF road network files"
+  homepage "http://ignitionrobotics.org"
+  url "http://gazebosim.org/distributions/ign-rndf/releases/ignition-rndf-0.1.2.tar.bz2"
+  sha256 "700bca1bb7af3ad02250dac9eef78a283742300239d5fc4494a18a8d419be307 "
+  
+  head "https://bitbucket.org/ignitionrobotics/ign-rndf", :branch => "default", :using => :hg
+
+  bottle do
+    root_url "http://gazebosim.org/distributions/ign-rndf/releases"
+    cellar :any
+    sha256 "0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF" => :el_capitan
+    sha256 "0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF" => :yosemite
+  end
+
+  depends_on "cmake" => :build
+  depends_on "ignition-math3"
+  depends_on "pkg-config" => :run
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "false"
```

Comment:
we should add a test that compiles an example program and runs it. see [sdformat5](https://github.com/osrf/homebrew-simulation/blob/master/sdformat5.rb#L42) for an example